Common/src/geometry/CMultiGridGeometry.cpp

```diff
@@ -435,258 +394,185 @@ CMultiGridGeometry::CMultiGridGeometry(CGeometry **geometry, CConfig *config_con
 
       nodes->SetnChildren_CV(iCoarsePoint_Complete, nChildren);
       nodes->SetnChildren_CV(iCoarsePoint, 0);
-
     }
+
   }
 
-  //  unsigned long iPointFree = nPointDomain-1;
-  //  iCoarsePoint = 0;
-  //
-  //  do {
```

Comment:
haven't seen a `do - while` in quite some time. 

Good to get rid of those commented code blocks